f5_openstack_agent/lbaasv2/drivers/bigip/resource_helper.py

```diff
@@ -244,3 +244,43 @@ def _collection(self, bigip):
                       "resource %s", self.resource_type)
             raise KeyError("No collection available for %s" %
                            (self.resource_type))
+
+    def get_stats(self, bigip,  name=None, partition=None, stats=[]):
```

Comment:
Call stats, statKeys, this way we know we are searching for something and not passing in statistics.  Nit-picky, but it might help readability.